CONTRIBUTING.md

```diff
@@ -66,7 +69,13 @@ Please ensure your commits are small and focused; they should tell a story of
 your change. This helps reviewers to follow your changes, especially for more
 complex changes.
 
-#### Signing
+#### Cryptographically Signing
+
+Vector requires all commits to be cryptographically signed as part of our
```

Comment:
Do we really? This is not something I've been doing and seems like a potentially large hurdle for accepting contributions.